publication-commons/src/main/java/no/unit/nva/publication/model/business/FileEntry.java

```diff
@@ -92,8 +92,8 @@ public static FileEntry fromDao(FileDao fileDao) {
         return (FileEntry) fileDao.getData();
     }
 
-    public static FileEntry importFileEntry(File file, SortableIdentifier identifier, UserInstance userInstance,
-                                            ImportSource importSource) {
+    public static FileEntry createFromImportSource(File file, SortableIdentifier identifier,
+                                                   UserInstance userInstance, ImportSource importSource) {
         var fileEntry = create(file, identifier, userInstance);
         fileEntry.setFileEvent(FileImportedEvent.create(userInstance.getUser(), Instant.now(), importSource));
         return fileEntry;
```

Comment:
We have to set that FileImportEvent both here and in importNew() methods cause that one is used in resourceService when creating transaction for file when persisting files and resource in single call.